drivers/i2c/i2c_ite_it8xxx2.c

```diff
@@ -240,15 +179,15 @@ static void i2c_reset(const struct device *dev)
 static void i2c_standard_port_timing_regs_400khz(uint8_t port)
 {
 	/* Port clock frequency depends on setting of timing registers. */
-	IT83XX_SMB_SCLKTS(port) = 0;
+	IT8XXX2_SMB_SCLKTS(port) = 0;
 	/* Suggested setting of timing registers of 400kHz. */
-	IT83XX_SMB_4P7USL = 0x6;
-	IT83XX_SMB_4P0USL = 0;
-	IT83XX_SMB_300NS = 0x1;
-	IT83XX_SMB_250NS = 0x2;
-	IT83XX_SMB_45P3USL = 0x6a;
-	IT83XX_SMB_45P3USH = 0x1;
-	IT83XX_SMB_4P7A4P0H = 0;
+	IT8XXX2_SMB_4P7USL = 0x3;
```

Comment:
Why are the timing registers changing?  This should be done in a separate PR.

Reply:
OK, separated.